Source/Core/Core/HW/WiimoteEmu/Camera.cpp

```diff
@@ -86,141 +101,100 @@ void CameraLogic::Update(const Common::Matrix44& transform)
 
   struct CameraPoint
   {
-    u16 x;
-    u16 y;
+    IRBasic::IRObject position;
     u8 size;
   };
 
   // 0xFFFFs are interpreted as "not visible".
-  constexpr CameraPoint INVISIBLE_POINT{0xffff, 0xffff, 0xff};
+  const CameraPoint INVISIBLE_POINT = {{0xffff, 0xffff}, 0xff};
```

Comment:
Perhaps changes should be made to keep this constexpr, otherwise technically a static runtime constructor may need to be emitted by compilers here.

Reply:
`Common::TVec2` isn't currently `constexpr`. I have some local changes `constexpr`ing it so I won't change that now.
I'll just eliminate `INVISIBLE_POINT`. It's only used in one place.